### PR TITLE
Document offline usage

### PR DESCRIPTION
**Pull request by axic**
@Denton-L can you double check the wording please?

### CODE REVIEW

**Comment by Denton-L** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ it will always deliver the latest version.
 
 ## Offline Usage
 
-Full offline usage is currently not supported because the compiler is always
-loaded via HTTP. If you clone/download the repository, use the
-`gh-pages` branch (otherwise you still have to build the application).
+The `gh-pages` branch always has the latest stable build of Browser-Solidity. It also contains a ZIP file with the entire build. Download it to use offline.
+
+Note: it contains the latest stable version of Solidity available at the time of the build. No other compiler versions are supported.
```
Comment (chriseth) [inline README.md]: Currently, it is still the latest nightly build.

Comment (axic) [inline README.md]: I'm being optimistic here :smiley:

**Comment by axic** [comment]
Changed it to reflect that it is a nightly build.

**Pull request merged**